lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -71,8 +72,28 @@ def add_arguments(self, parser):
             default=0,
             type=int,
         )
+        parser.add_argument(
+            '--estimate_first_attempted',
+            help='Use score data to estimate first_attempted timestamp. Values: (true|false|none)',
+            default='none',
+            choices={'true', 'false', 'none', 'True', 'False', 'None'}
```

Comment:
_>  If you're sure there's no use case for being able override the waffle flag once it's turned on in the database, I can make the false value mean "use the default," and the true value mean "unconditionally set the waffle switch."_

Yeah - I'd vote for simplicity and consistency in this case.  I don't anticipate needing that (YAGNI) - and if we ever do, it would be simple enough to change the code at that time.